mixer/adapter/stackdriver/config/google/api/label.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Google Inc.
```

Comment:
Is this temporary to copy google/api inside this config ?

Reply:
well, we need the source protos to be accessible to protoc. there isn't a very good way of doing that (for the status protos, we have to curl them down). moving forward, adapter authors may want to import from a variety of places (which may or may not be go-get-able). Putting them here to have a local copy is a quick solution.

We can't put them in `vendor`, as `dep` will overwrite them there. And baking in the dep to Mixer for a specific adapter seems wrong too.

I'm open to other suggestions.

Reply:
Can we commit this to https://github.com/istio/gogo-genproto/tree/master/googleapis/google/rpc?